src/Themes.tsx

```diff
@@ -25,19 +25,21 @@ interface ThemeHandler {
 type BaseComponentProps = ThemeProps & ThemeState & ThemeHandler;
 
 const BaseComponent: React.SFC<BaseComponentProps> = ({onSelectTheme, themes, theme}) => (
-    <FlexRow>
-        {themes.map((th, i) => (
-            <Button
-                id={`theme-selection-button-${th.name}`}
-                selected={th === theme}
-                key={i}
-                onClick={() => onSelectTheme(th)}>
-                {th.name}
-            </Button>
-        )).toArray()}
-        <FillingDiv />
-        <Border>|</Border>
-    </FlexRow>
+    <div>
+        <ButtonContainer>
+            {themes.map((th, i) => (
+                <Button
+                    id={`theme-selection-button-${th.name}`}
+                    selected={th === theme}
+                    themeColor={th.addonOptions?.themeColor}
```

Comment:
I'm not sure `?` will be broken when `this.addonOptions === undefined` or not
my you try this case with `undefined` addonOptions?